qa/suites/fs/nfs/nfs-version/latest-nfs.yaml

```diff
@@ -0,0 +1,2 @@
+nfs:
+  vers: latest
```

Comment:
I like the idea of testing with `vers=` option unset, even if it's the same as `4.2`. That may change someday or the nfs linux driver may soon "add" things beyond 4.2 without adding an explicit version bump.

IOW, please add a `4.2.yaml` and a `unset.yaml`.